docs/Principles/modeling-principles.md

```diff
@@ -0,0 +1,27 @@
+# Modeling Principles
+
+## Conforms to Schema
+The Monarch Biolink Specification is an implementation of the Biolink Model.
```

Comment:
Is this document itself the Monarch Biolink Specification, or is that another document? If the latter, should add link to it.